packages/world/src/EntityId.sol

```diff
@@ -83,7 +83,7 @@ library EntityIdLib {
 
   function requireConnected(EntityId self, Vec3 otherCoord) internal view returns (Vec3, Vec3) {
     Vec3 selfCoord = self._getPosition();
-    require(selfCoord.inSurroundingCube(otherCoord, MAX_ENTITY_INFLUENCE_HALF_WIDTH), "Entity is too far");
+    require(selfCoord.inSphere(otherCoord, MAX_ENTITY_INFLUENCE_HALF_WIDTH), "Entity is too far");
```

Comment:
have you tested with the client to see if it causes any issues when trying to mine diagonally?

Reply:
not yet! will do soon